utils/sample_qc.py

```diff
@@ -364,91 +364,109 @@ def infer_sex(
 
 
 def get_sex_expr(
-    chr_x_ploidy: hl.expr.NumericExpression,
```

Comment:
why remove these two expression params?

Reply:
I changed the code to take in a ht (annotated with ploidy and with the output of hail's `impute_sex`) for the `get_ploidy_cutoffs` subfunction, so I removed the extra parameters

Reply:
ah yeah... hmm why?
I think that returning expressions is a better / more modular way of structuring code.